external-docs/docs/guides/shape-data.md

```diff
@@ -1,3 +1,241 @@
+    * the `Year` and `Location` _dimension_ columns define the subset of the population that the row's observed value covers, i.e. each row describes the number of _Arthur's Bakes_ stores in a particular UK city in a given year.
+3. Every row has an _observed values_ column containing the measured value of some property.
+    * the `Value` column holds the observed values here.
+4. Each piece of information describing the _observed value_ has its own _attribute_ column.
+    * the `Status` _attribute_ column contains information describing the state of the observed value itself.
+    * Note that _attributes_ should only describe the observed value and must not identify any subset of the population.  
+
+## Canonical Shape
+
+The canonical shape extends the [common structure](#common-structure) by requiring that **each row** has a _measures_ column and a _units_ column; these columns define the measure and unit (of measure) for each row.
+
+| Year | Location  | Value |      Status |                    Measure |  Unit |
+|:-----|:----------|------:|------------:|---------------------------:|------:|
+| 2022 | London    |    35 | Provisional | Number of 'Arthur's Bakes' | Count |
+| 2021 | Cardiff   |    26 |       Final | Number of 'Arthur's Bakes' | Count |
+| 2020 | Edinburgh |    90 |       Final | Number of 'Arthur's Bakes' | Count |
+| 2021 | Belfast   |     0 |       Final | Number of 'Arthur's Bakes' | Count |
+
+The simplest [qube-config.json](./qube-config.md#configuration) we can define for this data set is:
+
+```json
+{
+    "$schema": "http://purl.org/csv-cubed/qube-config/v1.0",
+    "title": "'Arthur's Bakes' stores in UK cities from 2020 to 2022",
+    "description": "The number of 'Arthurs' Bakes' stores in cities across the UK between 2020 and 2022.",
+    "creator": "HM Revenue & Customs",
+    "publisher": "HM Revenue & Customs",
+    "columns": {
+        "Year": {
+            "type": "dimension"
+        },
+        "Location": {
+            "type": "dimension"
+        },
+        "Value": {
+            "type": "observations"
+        },
+        "Status": {
+            "type": "attribute"
+        },
+        "Measure": {
+            "type": "measures"
+        },
+        "Unit": {
+            "type": "units"
+        }
+    }
+}
+```
+
+It is possible to use the [conventional-first approach](./qube-config.md#convention-first-method) to generate a valid [canonical shape](#canonical-shape) cube without defining a [qube-config.json](./qube-config.md#configuration) at all. Just ensure that your columns use the [conventional column names](./qube-config.md#conventional-column-names) appropriate to their type.
+
+### Multiple Measures
+
+One of the benefits of the canonical shape is that it is relatively straightforward to add new measure types and unit types; all that you have to do is add additional rows to your data set with the appropriate units and measures present.
```

Comment:
What will involve doing the same with the "Common Structure" approach? E.g. "In contrast, the Common Structure involves ........."